marconi-chain-index/src/Marconi/ChainIndex/CLI.hs

```diff
@@ -199,3 +164,45 @@ epochStakepoolSizeDbPath o = if optionsDisableStakepoolSize o then Nothing else
 
 mintBurnDbPath :: Options -> Maybe FilePath
 mintBurnDbPath o = if optionsDisableMintBurn o then Nothing else Just (optionsDbPath o </> mintBurnDbName)
+
+-- * Common CLI parsers
+
+commonSocketPath :: Opt.Parser String
```

Comment:
Why the prefix "common"? If it's because that CLI parser is common to `marconi-sidechain`, then I think there is no need to do that. Just `socketPathParser` is enough. Is will be to client libraries like `marconi-sidechain` to choose their own parsers, right?

Reply:
It's to say that the flag (e.g `-s`) is also the common one, not only the field value parser. If you read the code and see Cli.commonSocketPath you'll know what it would be parsing, e.g Cli.parseSocketPath parses socket path but with which name, is it the common one or something else?

Reply:
Do you mean that common means "common between all indexers? Maybe mention common to what?

> If you read the code and see Cli.commonSocketPath you'll know what it would be parsing, e.g Cli.parseSocketPath parses socket path but with which name, is it the common one or something else?

Not sure I understand.

Reply:
Then, I would add a comment saying something like: "Common CLI parsers for all CLI programs derived from this project."